YoutubeDownloader/Views/Components/DashboardView.axaml

```diff
@@ -64,6 +64,8 @@
                     VerticalAlignment="Center"
                     Command="{Binding ShowAuthSetupCommand}"
                     Foreground="{DynamicResource MaterialDarkForegroundBrush}"
+                    IsVisible="{OnPlatform False,
+                                           Windows=True}"
```

Comment:
Even though the authentication _should_ work on other platforms, we'll limit it to Windows for now